### ISSUE TITLE
Detecting origins

### ISSUE THREAD

**Issue by mkungla** [newren/git-filter-repo#106]
```
git clone --origin something git@github.com:org/repo.git
git filter-repo \
  --source ./repo \
  --target ./new
```

That is false positive when target is cloned using `--origin`

```
Aborting: Refusing to overwrite repo history since this does not
look like a fresh clone.
  (expected one remote, origin)
To override, use --force.

```

**Comment by newren** [comment]
Yeah, I should add a comment to the documentation somewhere instructing people not to use `--origin` when they're about to rewrite the history with filter-repo.

**Comment by newren** [comment]
Addressed by commit a238e3b7e6 (git-filter-repo.txt: discourage use of random clone flags, 2020-06-13), to suggest that people don't use any options to clone other than --no-local, --bare, and --mirror when working with filter-repo.
